### PR TITLE
Add bobbygeeze as a CF member

### PR DESCRIPTION
**Pull request by bobbygeeze**
I am a new docs writer for VMware and I am requesting to become an  approver for all doc areas.

### CODE REVIEW

**Comment by bobbygeeze** [comment]
I am a new docs writer from VMware and am requesting to become an approver for all docs areas.

This affects 3 working groups and requires approval from the tech leads in each working group:
* App Runtime Platform - @ameowlia
* App Runtime Interfaces - @gerg
* Foundational Infrastructure - @beyhan and/or @rkoster

**Review by ameowlia — APPROVED** [review]

**Comment by ameowlia** [comment]
Closing in favor of this [duplicate PR](https://github.com/cloudfoundry/community/pull/531).

**Pull request closed without merging**
